### PR TITLE
Custom metrics for observability updates 2.2

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/10884
open-cluster-management/backlog#10538
open-cluster-management/backlog#10827

### CODE REVIEW

**Comment by dockerymick** [comment]
@haoqing0110 thank you for your changes in 2.3. I was working on a different issue, where I needed to update 2.2. I noticed that your commits from 2.3 were pulled in to 2.2, so i reverted the changes in the 2.2_stage. May you confirm if the content for cert_mgmt_ingress.adoc is accurate? 

Or, should your changes from 2.3_stage also be applied to 2.2_stage?

**Comment by haoqing0110** [comment]
@dockerymick the PR https://github.com/open-cluster-management/rhacm-docs/pull/1945 is only for 2.3_stage.

File: security/cert_mgmt_ingress.adoc [hunk 1]
```diff
@@ -55,7 +56,7 @@ CN = management-ingress  # Common name.
 subjectAltName = @alt_names # Subject alternative names
 
 [ alt_names ]        # Subject alternative names
-DNS.1 = management-ingress
+DNS.1 = multicloud-console.apps.grchub2.dev08.red-chesterfield.com
```
Comment (haoqing0110) [inline security/cert_mgmt_ingress.adoc]: Seems this line is incorrect, DNS.1 and DNS.2 are the same

**Comment by dockerymick** [comment]
@haoqing0110 thank you so much for catching that!

**Pull request merged**